README.md

```diff
@@ -1,10 +1,10 @@
-# Webapp Runner [![Build Status](https://travis-ci.org/heroku/webapp-runner.svg?branch=master)](https://travis-ci.org/heroku/webapp-runner) [![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.heroku/webapp-runner/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.heroku/webapp-runner)
+# Webapp Runner [![Build Status](https://travis-ci.com/heroku/webapp-runner.svg?branch=main)](https://travis-ci.com/heroku/webapp-runner) [![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.heroku/webapp-runner/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.heroku/webapp-runner)
```

Comment:
will the `.com` mean it's inaccessible for the public?

Reply:
Nope, Travis migrated all the open source repositories to their `.com` domain some time ago. This step just skips one manual "yes, it's okay to redirect me" page.

Reply:
Travis are partway through a move of all open source projects on .org to .com:
https://docs.travis-ci.com/user/migrate/open-source-repository-migration/

The open source project .com URLs still load for me in a private browsing session.